rotkehlchen/db/updates.py

```diff
@@ -372,7 +372,7 @@ def check_for_updates(self, updates: Sequence[UpdateType] = tuple(UpdateType)) -
     def _check_for_last_version(cursor: 'DBCursor', update_type: UpdateType) -> int:
         """This method checks the database for the last local version of `update_type`"""
         found_version = cursor.execute(
-            'SELECT value FROM key_value_cache WHERE name=?',
+            'SELECT value FROM settings WHERE name=?',
```

Comment:
why this change? Weren't they suppose to be in the value cache table?

Reply:
We kept them in the settings table in the last PR https://github.com/rotki/rotki/pull/7178/#discussion_r1438117221